draft-tuexen-opsawg-pcapng.xml

```diff
@@ -2090,15 +2090,21 @@ Section Header
 
             <t hangText="0x5a4e574b:"><vspace blankLines="0"/>ZigBee NWK Key.
             Network Key as described in the <eref target="https://zigbeealliance.org/">
-            ZigBee Specification</eref> R21 section 4.2.2. The NWK Key is a 16 octet
-            AES-128 key used to secure NWK Level frames within a single PAN.
+            ZigBee Specification</eref> 05-3473-21 (R21) section 4.2.2. The NWK Key
+            is a 16 octet binary AES-128 key used to secure NWK Level frames
+            within a single PAN. If and when the NWK Key changes a new DSB will
+            contain the new NWK Key.
```

Comment:
So an NWK Key entry applies to all packets in packet blocks between the DSB in which an NWK Key appears and the next DSB in which an NWK Key appears?

Also, if, for example, a capture includes traffic from multiple PANs, what indicates the PAN to which a given NWK Key applies?

Reply:
In essence yes. The purpose of the DSB is to replace the Pre-configured Keys UAT available at the time of capture but very likely different later when the PCAPNG file is opened. However you raise a good point because the key when looking up the nwk hints is the PANID. So, when loading the file the hash table lookup will need the PANID to locate the nwk_keyring. I'll update the document.